aten/src/ATen/xpu/profiler/XPUActivityApi.h

```diff
@@ -0,0 +1,71 @@
+#pragma once
+
+#include <ATen/xpu/profiler/XPUActivityBuffer.h>
+
+#include <atomic>
+#include <functional>
+#include <mutex>
+#include <set>
+
+#include "XPUProfilerMacros.h"
```

Comment:
```suggestion
#include <ATen/xpu/profiler/XPUProfilerMacros.h>
```
and move it to line 4.

Reply:
ok